src/front-end/src/components/MentorForm.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import { Form, Button, Input, DatePicker, Tooltip, Select } from 'antd';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { InfoCircleOutlined, QuestionCircleOutlined } from '@ant-design/icons';
+import TimezonePicker from 'react-bootstrap-timezone-picker';
+import 'react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css';
+
+const { RangePicker } = DatePicker;
+
+const { Option } = Select;
+
+class MentorForm extends Component {
+  render() {
+    return (
+    <div className="MentorForm"> 
+      <Form>
+        <Form.Item
+          // TEST ITEM (will be deleted):
+          label="Organization"
+          name="Organization"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input
+            placeholder="Please enter your organization"
+            prefix={
+              // tooltip test:
+              <Tooltip title="Organization">
+                <InfoCircleOutlined style={{ color: 'rgba(0,0,0,.45)' }} />
+              </Tooltip>
+            }
+          />
+        </Form.Item>
+        <Form.Item
+          label="Name"
+          name="nameInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your name" />
+        </Form.Item>
+        <Form.Item
+          label="Email"
+          name="emailInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your email" />
+        </Form.Item>
+        <Form.Item
+          label={
+            <span>
+              Timeframe Seeking Mentorship&nbsp;
+              <Tooltip title="Please pick an estimated duration for how long you would like to use the platform">
+                <QuestionCircleOutlined />
+              </Tooltip>
+            </span>
+          }
+          name="selectedStartDate"
+          rules={[{ required: true, message: 'This field is required' }]}
+        >
+          <RangePicker picker="month" />
+        </Form.Item>
+        <Form.Item

```

Comment:
Same comment as above, this field can now be removed.

Reply:
done